rust/hyperlane-core/src/db/hyperlane_db.rs

```diff
@@ -243,7 +232,7 @@ impl HyperlaneDB {
         let existing_payment = self.retrieve_gas_payment_for_message_id(*message_id)?;
         let total = existing_payment + payment;
 
-        info!(message_id=?message_id, gas_payment_amount=?payment, new_total_gas_payment=?total, "Storing gas payment");
+        info!(?message_id, gas_payment_amount=?payment, new_total_gas_payment=?total, "Storing gas payment");
```

Comment:
We use `message_id` here and `id` in one of the logs above, let's make them consistent. Your pick